### DISCUSSION TITLE
help me please

### DISCUSSION THREAD

**Discussion by valvaoun** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#310] [General]
[      script:oxmysql] Error: Unable to establish a connection to the database! [AUTH_SWITCH_PLUGIN_ERROR]
[      script:oxmysql] Server requests authentication using unknown plugin auth_gssapi_client. See TODO: add plugins doco here on how to configure or author authentication plugins.
[     script:soz-core] [2024-03-16T01:06:11.175Z] [Prisma] [quaint::pooled] Starting a mysql pool with 13 connections.
[     script:soz-core] [2024-03-16T01:06:11.180Z] [Prisma] [query_engine] Encountered error during initialization:
[     script:soz-core] [2024-03-16T01:06:11.188Z] Error on executing step start in method onStart of provider wr PrismaClientInitializationError: Error querying the database: Unknown authentication plugin `auth_gssapi_client'. PrismaClientInitializationError: Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core]     at kr.<anonymous> (@soz-core/build/server.js:2:599050)
[     script:soz-core]     at kr.emit (node:events:394:28)
[     script:soz-core]     at addChunk (node:internal/streams/readable:315:12)
[     script:soz-core]     at readableAddChunk (node:internal/streams/readable:289:9)
[     script:soz-core]     at kr.Readable.push (node:internal/streams/readable:228:10)
[     script:soz-core]     at kr._pushBuffer (@soz-core/build/server.js:2:580037)
[     script:soz-core]     at kr._transform (@soz-core/build/server.js:2:579867)
[     script:soz-core]     at kr.Transform._write (node:internal/streams/transform:184:23)
[     script:soz-core]     at writeOrBuffer (node:internal/streams/writable:389:12)
[     script:soz-core]     at _write (node:internal/streams/writable:330:10)
[     script:soz-core] [2024-03-16T01:06:11.189Z] [Prisma] [field.findFirst] 
[     script:soz-core] Invalid `prisma.field.findFirst()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core] [2024-03-16T01:06:11.189Z] [Prisma] [playerVehicle.deleteMany] 
[     script:soz-core] Invalid `prisma.playerVehicle.deleteMany()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core] [2024-03-16T01:06:11.189Z] [Prisma] [field.findFirst] 
[     script:soz-core] Invalid `prisma.field.findFirst()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core] [2024-03-16T01:06:11.189Z] [Prisma] [field.findFirst] 
[     script:soz-core] Invalid `prisma.field.findFirst()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core] [2024-03-16T01:06:11.190Z] Error on executing step start in method onStart of provider Fd PrismaClientInitializationError: 
[     script:soz-core] Invalid `prisma.playerVehicle.deleteMany()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'. PrismaClientInitializationError: 
[     script:soz-core] Invalid `prisma.playerVehicle.deleteMany()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core]     at ec.handleRequestError (@soz-core/build/server.js:2:691187)
[     script:soz-core]     at ec.handleAndLogRequestError (@soz-core/build/server.js:2:690309)
[     script:soz-core]     at ec.request (@soz-core/build/server.js:2:689919)
[     script:soz-core]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[     script:soz-core]     at async Fd.onStart (@soz-core/build/server.js:2:1431827)
[     script:soz-core]     at async Object.a [as method] (@soz-core/build/server.js:2:989523)
[     script:soz-core]     at async Promise.all (index 19)
[     script:soz-core]     at async Yt.trigger (@soz-core/build/server.js:2:989396)
[     script:soz-core]     at async e.start (@soz-core/build/server.js:2:1001043)
[     script:soz-core]     at async Function.create (@soz-core/build/server.js:2:1000603)
[     script:soz-core] [2024-03-16T01:06:11.190Z] Error on executing step start in method init of provider rh PrismaClientInitializationError: 
[     script:soz-core] Invalid `prisma.field.findFirst()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'. PrismaClientInitializationError: 
[     script:soz-core] Invalid `prisma.field.findFirst()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core]     at ec.handleRequestError (@soz-core/build/server.js:2:691187)
[     script:soz-core]     at ec.handleAndLogRequestError (@soz-core/build/server.js:2:690309)
[     script:soz-core]     at ec.request (@soz-core/build/server.js:2:689919)
[     script:soz-core]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[     script:soz-core]     at async _u.createField (@soz-core/build/server.js:2:1381579)
[     script:soz-core]     at async rh.init (@soz-core/build/server.js:2:1478394)
[     script:soz-core]     at async Object.a [as method] (@soz-core/build/server.js:2:989523)
[     script:soz-core]     at async Promise.all (index 17)
[     script:soz-core]     at async Yt.trigger (@soz-core/build/server.js:2:989396)
[     script:soz-core]     at async e.start (@soz-core/build/server.js:2:1001043)
[     script:soz-core] [2024-03-16T01:06:11.190Z] Error on executing step start in method onInit of provider Xg PrismaClientInitializationError: 
[     script:soz-core] Invalid `prisma.field.findFirst()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'. PrismaClientInitializationError: 
[     script:soz-core] Invalid `prisma.field.findFirst()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core]     at ec.handleRequestError (@soz-core/build/server.js:2:691187)
[     script:soz-core]     at ec.handleAndLogRequestError (@soz-core/build/server.js:2:690309)
[     script:soz-core]     at ec.request (@soz-core/build/server.js:2:689919)
[     script:soz-core]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[     script:soz-core]     at async _u.createField (@soz-core/build/server.js:2:1381579)
[     script:soz-core]     at async Xg.onInit (@soz-core/build/server.js:2:1555389)
[     script:soz-core]     at async Object.a [as method] (@soz-core/build/server.js:2:989523)
[     script:soz-core]     at async Promise.all (index 20)
[     script:soz-core]     at async Yt.trigger (@soz-core/build/server.js:2:989396)
[     script:soz-core]     at async e.start (@soz-core/build/server.js:2:1001043)
[     script:soz-core] [2024-03-16T01:06:11.191Z] Error on executing step start in method init of provider Op PrismaClientInitializationError: 
[     script:soz-core] Invalid `prisma.field.findFirst()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'. PrismaClientInitializationError: 
[     script:soz-core] Invalid `prisma.field.findFirst()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core]     at ec.handleRequestError (@soz-core/build/server.js:2:691187)
[     script:soz-core]     at ec.handleAndLogRequestError (@soz-core/build/server.js:2:690309)
[     script:soz-core]     at ec.request (@soz-core/build/server.js:2:689919)
[     script:soz-core]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[     script:soz-core]     at async _u.createField (@soz-core/build/server.js:2:1381579)
[     script:soz-core]     at async Op.init (@soz-core/build/server.js:2:1453478)
[     script:soz-core]     at async Object.a [as method] (@soz-core/build/server.js:2:989523)
[     script:soz-core]     at async Promise.all (index 33)
[     script:soz-core]     at async Yt.trigger (@soz-core/build/server.js:2:989396)
[     script:soz-core]     at async e.start (@soz-core/build/server.js:2:1001043)
[     script:soz-core] [2024-03-16T01:06:11.263Z] [Prisma] [quaint::pooled] Starting a mysql pool with 13 connections.
[     script:soz-core] [2024-03-16T01:06:11.269Z] [Prisma] [query_engine] Encountered error during initialization:
[     script:soz-core] [2024-03-16T01:06:11.275Z] [Prisma] [playerVehicle.findMany] 
[     script:soz-core] Invalid `prisma.playerVehicle.findMany()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core] [2024-03-16T01:06:11.275Z] error in soft-pound-check: PrismaClientInitializationError: 
[     script:soz-core] Invalid `prisma.playerVehicle.findMany()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core] PrismaClientInitializationError: 
[     script:soz-core] Invalid `prisma.playerVehicle.findMany()` invocation:
[     script:soz-core] 
[     script:soz-core] 
[     script:soz-core] Error querying the database: Unknown authentication plugin `auth_gssapi_client'.
[     script:soz-core]     at ec.handleRequestError (@soz-core/build/server.js:2:691187)
[     script:soz-core]     at ec.handleAndLogRequestError (@soz-core/build/server.js:2:690309)
[     script:soz-core]     at ec.request (@soz-core/build/server.js:2:689919)
[     script:soz-core]     at processTicksAndRejections (node:internal/process/task_queues:96:5)
[     script:soz-core]     at async WE.loop (@soz-core/build/server.js:2:1751751)
[     script:soz-core]     at async @soz-core/build/server.js:2:1013805
[     script:soz-core]     at async @soz-core/build/server.js:2:1016913
[     script:soz-core]     at async @soz-core/build/server.js:2:1014240
[     script:soz-core]     at async Object.callback (@soz-core/build/server.js:2:997285)
(node:4036) UnhandledPromiseRejectionWarning: Error: soz-bank was unable to execute a query!
    Server requests authentication using unknown plugin auth_gssapi_client. See TODO: add plugins doco here on how to configure or author authentication plugins.
    SELECT * FROM bank_accounts []
    at query_default (@oxmysql/dist/server/build.js:16101:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
(Use `FXServer --trace-warnings ...` to show where the warning was created)
(node:4036) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:4036) UnhandledPromiseRejectionWarning: Error: soz-bank was unable to execute a query!
    Server requests authentication using unknown plugin auth_gssapi_client. See TODO: add plugins doco here on how to configure or author authentication plugins.
    SELECT * FROM invoices WHERE payed = false AND refused = false []
    at query_default (@oxmysql/dist/server/build.js:16101:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
(node:4036) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:4036) UnhandledPromiseRejectionWarning: Error: soz-inventory was unable to execute a query!
    Server requests authentication using unknown plugin auth_gssapi_client. See TODO: add plugins doco here on how to configure or author authentication plugins.
    DELETE FROM storages WHERE NAME IN (SELECT name FROM storages LEFT OUTER JOIN player_vehicles on storages.owner = player_vehicles.plate where storages.`type`='trunk' AND player_vehicles.citizenid IS NULL) []
    at query_default (@oxmysql/dist/server/build.js:16101:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
(node:4036) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 3)
(node:4036) UnhandledPromiseRejectionWarning: Error: soz-inventory was unable to execute a query!
    Server requests authentication using unknown plugin auth_gssapi_client. See TODO: add plugins doco here on how to configure or author authentication plugins.
    SELECT * FROM storages []
    at query_default (@oxmysql/dist/server/build.js:16101:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
(node:4036) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 4)
(node:4036) UnhandledPromiseRejectionWarning: Error: soz-upw was unable to execute a query!
    Server requests authentication using unknown plugin auth_gssapi_client. See TODO: add plugins doco here on how to configure or author authentication plugins.
    SELECT * FROM upw_facility WHERE identifier = ? ["pm1"]
    at query_default (@oxmysql/dist/server/build.js:16101:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
(node:4036) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 5)
(node:4036) UnhandledPromiseRejectionWarning: Error: soz-pawl was unable to execute a query!
    Server requests authentication using unknown plugin auth_gssapi_client. See TODO: add plugins doco here on how to configure or author authentication plugins.
    SELECT * FROM field WHERE owner = 'pawl' []
    at query_default (@oxmysql/dist/server/build.js:16101:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
(node:4036) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 6)

**Comment by rayanio** [comment]
Try checking the version of your MySQL server, to make sure it supports the authentication plugin you're using (I suppose it's _auth_gssapi_client_). If it doesn't, consider upgrading to a version that does.
